pkg/commands/secretstoreentry/secretstoreentry_test.go

```diff
@@ -138,6 +181,11 @@ func TestCreateSecretCommand(t *testing.T) {
 
 			opts.APIClient = mock.APIClient(testcase.api)
 
+			// Tests generate their own signing keys, which won't match
+			// the hardcoded value.  Loosen the validation checks.
+			os.Setenv("FASTLY_USE_API_SIGNING_KEY", "1")
+			defer os.Unsetenv("FASTLY_USE_API_SIGNING_KEY")
```

Comment:
Could [`testing.Setenv`](https://pkg.go.dev/testing#T.Setenv) be used here?

Reply:
I did not know about `testing.Setenv`!  TIL, will definitely do that.